core/src/com/unciv/logic/trade/TradeLogic.kt

```diff
@@ -155,8 +159,6 @@ class TradeLogic(val ourCivilization:Civilization, val otherCivilization: Civili
 
         for (offer in currentTrade.theirOffers.filter { it.type == TradeType.Treaty })
```

Comment:
Please add a documentation note here, if I was reading this for the first time I would be confused why it's not mirrored